### ISSUE TITLE
[OpenTK] Leaks when closing application

### ISSUE THREAD

**Issue by mrhelmut** [mono/MonoGame#4412]
```Game.Exit()``` results in the following console output (debug):

```
XInputJoystick leaked, did you forget to call Dispose()?
NativeWindowBase leaked, did you forget to call Dispose()?
[Warning] INativeWindow leaked (OpenTK.Platform.Windows.WinGLNative). Did you forget to call INativeWindow.Dispose()?
[Warning] Resource leaked: OpenTK.Platform.Windows.WinRawInput.
[Warning] Failed to release device context 184620535. Windows error: 0.
NativeWindowBase leaked, did you forget to call Dispose()?
[Warning] INativeWindow leaked (OpenTK.Platform.Windows.WinGLNative). Did you forget to call INativeWindow.Dispose()?
[Warning] {0} leaked, did you forget to call Dispose()?
[OpenTK] OpenTK.Platform.Factory+UnsupportedPlatform leaked with 0 live resources, did you forget to call Dispose()?
[OpenTK] OpenTK.Platform.Windows.WinFactory leaked with 5 live resources, did you forget to call Dispose()?
OpenTK.Platform.Factory leaked, did you forget to call Dispose()?
```

Seeing joystick driver leaking makes me think of an issue described in #4252: on MacOS, after closing an application, GamePads don't work anymore unless they are unplugged & re-plugged.

P.S.: this is the Windows output, I'm not seeing on MacOS. :grey_question:

**Comment by cra0zy** [comment]
Bright side: #4399

**Comment by mrhelmut** [comment]
Fixed by #4755
